ocaml/xapi/xapi_host.ml

```diff
@@ -2791,13 +2795,16 @@ let write_uefi_certificates_to_disk ~__context ~host =
         Db.Pool.set_uefi_certificates ~__context
           ~self:(Helpers.get_pool ~__context)
           ~value:disk_uefi_certs_tar
-  | true ->
-      let@ path = with_empty_dir !Xapi_globs.varstore_dir in
-      (* get from pool for consistent results across hosts *)
-      let pool_uefi_certs =
-        Db.Pool.get_uefi_certificates ~__context
-          ~self:(Helpers.get_pool ~__context)
+  | true, "" ->
+      (* When overriding certificates and user hasn't yet specified a value,
```

Comment:
This comment somehow implies that the user is going to specify a value, but to me, `override_uefi_certs = true`  doesn't mean "the user will override the certs", but rather "we leave the ability to override the certs if needed". Hence, rather than a transitional state, I see this state where we keep the symlink as XCP-ng's default state, where the certs remain in sync with the defaults unless the user installs new certs to override them.